src/main/kotlin/no/nav/k9/domene/repository/StatistikkRepository.kt

```diff
@@ -375,57 +375,27 @@ class StatistikkRepository(
         }
     }
 
-    fun hentReservasjoner(): List<Reservasjon> {
-        val reservasjoner = using(sessionOf(dataSource)) {
+    fun hentTiderFraOpprettetBehandlingTilFørsteReservasjon(): List<Pair<LocalDateTime, LocalDateTime>> {
+        val tiderFraOpprettetBehandlingTilFørsteReservasjon = using(sessionOf(dataSource)) {
             it.run(
-                queryOf(
-                    "select id, opprettet from reservasjon where opprettet is not null"
-                ).map { rad ->
-                    Reservasjon(
-                        reservertTil = null,
-                        reservertAv = "",
-                        flyttetAv = "",
-                        flyttetTidspunkt = null,
-                        begrunnelse = null,
-                        oppgave = UUID.fromString(rad.string("id")),
-                        opprettet = rad.localDateTimeOrNull("opprettet")
+                queryOf("""
+                    select b.opprettet behandling_opprettet, r.opprettet reservasjon_opprettet
+                    from reservasjon r
+                        inner join behandling b on r.id = b.ekstern_referanse
+                        inner join oppgave o on r.id = o.id
+                    where o.data -> 'aktiv' = 'true'
```

Comment:
Det er ikke sikkert at det har så mye å si for utfallet, men vi bør utelate alle oppgaver der det allerede finnes en reservasjon uten opprettet-timestamp.
F.eks i tilfeller der en oppgave har fått utført litt arbeid tidligere, men en ny reservasjonen opprettes etter at endringen din er lansert.